site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,459 @@
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/DbkcylGpILjOzOnxREXl.png",
+       alt="SXG Validator showing cache information including URL",
+       width="506", height="229" %}
+
+Once these tests are complete, go to [Test History](https://app.webpagetest.org/ui/wpt/testhistory), select the two tests, and click Compare:
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/ElSV7nAWCWV0Yjf7VC2o.png",
+       alt="Test History with two tests checked and the Compare button highlighted",
+       width="509", height="202" %}
+
+Append `?medianMetric=LCP` to the compare URL so that the run with median LCP is selected for each side of the comparison. (The default is median by Speed Index.)
+
+To compare waterfalls, expand the Waterfall Opacity section and drag the slider. To view the video, click Adjust Filmstrip Settings, scroll down inside that dialog, and click View Video.
+
+## Debug
+
+If the WebPageTest is showing that the HTML is being prefetched, then it has succeeded in all the steps of the pipeline; you may skip to the [Optimize](#optimize) section to learn how to further improve LCP. Otherwise, you'll need to find out where in the pipeline it failed and why; see the following sections for how.
+
+### Publishing
+
+Make sure your pages are being generated as signed exchanges.
+
+{% Aside %}
+Your site will not serve SXG to users, even on SXG-capable browsers. They're meant for SXG-capable crawlers, to enable prefetch. (Serving SXGs to users who visit the site directly only adds overhead over a perfectly cromulent web page.) Thus, to debug, you need to pretend to be a crawler.
+{% endAside %}
+
+The easiest way is to use the SXG Validator Chrome extension:
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/zUqSkRSLNkF26gjyymEo.png",
+       alt="SXG Validator showing a check mark (✅) and a Content Type of application/signed-exchange;v=b3",
+       width="600", height="281" %}
+
+The extension fetches the current URL again, but with an Accept header that says it prefers the SXG version. If you see a check mark (✅), that means an SXG was returned; you can skip to the [Indexing](#indexing) section.
+
+If you see a cross mark (❌), that means an SXG wasn't returned:
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/4T3la9HtoRDz8SK7lWGi.png",
+       alt="SXG Validator showing a cross mark (❌) and a Content Type of text/html",
+       width="483", height="162" %}
+
+If Cloudflare ASX is enabled, then the most likely reason for a cross mark (❌) is because a cache control response header prevents it. ASX looks at headers with the following names:
+  - `Cache-Control`
+  - `CDN-Cache-Control`
+  - `Surrogate-Control`
+  - `Cloudflare-CDN-Cache-Control`
+
+If any of these headers contains any of the following header values, it will prevent an SXG from being generated:
+  - `private`
+  - `no-store`
+  - `no-cache`
+  - `s-maxage` less than 120, if present, or else `max-age` less than 120
```

Comment:
```suggestion
If Cloudflare ASX is enabled, then the most likely reason for a cross mark (❌) is because a cache control response header prevents it. ASX looks at headers with the following names:
- `Cache-Control`
- `CDN-Cache-Control`
- `Surrogate-Control`
- `Cloudflare-CDN-Cache-Control`

If any of these headers contains any of the following header values, it will prevent an SXG from being generated:
- `private`
- `no-store`
- `no-cache`
- `s-maxage` less than 120, if present, or else `max-age` less than 120
```